akka-docs/src/test/scala/docs/stream/IntegrationDocSpec.scala

```diff
@@ -475,4 +501,49 @@ class IntegrationDocSpec extends AkkaSpec(IntegrationDocSpec.config) {
       "after: J"))
   }
 
+  "illustrate use of actorRefWithAck along with stream" in {
+    //#actorRefWithAck
+    import MessageProcessor._
+
+    val messageProcessorActor =
+      system.actorOf(Props[MessageProcessor], "messageProcessor")
+
+    val source: Source[Int, NotUsed] =
+      Source.fromIterator(() ⇒ (0 to 10).toIterator)
+
+    val flow = Flow[Int].map(ProcessMessage)
+
+    val sink =
+      Sink.actorRefWithAck(messageProcessorActor, Init, Ack, Completed)
+
+    source.via(flow).to(sink).run()
+    //#actorRefWithAck
+  }
```

Comment:
The corresponding Java sample already existed?

Reply:
Yeah `actorRefWithAck` I documented in https://github.com/akka/akka/pull/24854

Reply:
👍